client/webserver/site/src/js/wallets.ts

```diff
@@ -727,30 +847,58 @@ export default class WalletsPage extends BasePage {
     }
   }
 
+  /* populateMaxSend populates the amount field with the max amount the wallet
+     can send. The max send amount can be the maximum amount based on our
+     pre-estimation or the asset's wallet balance.
+  */
+  async populateMaxSend () {
+    const page = this.page
+    const asset = app().assets[this.selectedAssetID]
+    if (!asset) return
+    // Populate send amount with max send value and ensure we don't check
+    // subtract checkbox for assets that don't have a withdraw method.
+    if ((asset.wallet.traits & traitWithdrawer) === 0) {
+      const maxSend = parseFloat(page.maxSend.textContent || '0')
```

Comment:
Parsing values from `textContent` is dangerous with translations. You'll need to cache this value on `WalletPage` somehow.